jenkins-design-language/src/js/components/material-ui/svg-icons/action/accessibility.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import pure from 'recompose/pure';
```

Comment:
Well we don't need everything in the material-ui, the icons are just a small part of it, secondly, if we have them in our repo, we can control if/when we update the icons and maybe add our own. That doesn't mean we can't change anything, I just explained my rationale for what I did, if you think it is better to remove the recompose/pure dep and/or change the name of the icons to not include the category name, it is totally doable and the benefit of hosting the icons ourselves, the only downside will be when we will want to update the icons. What do you think?